eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -177,23 +167,26 @@ foreach ($package in $packagesForToc.Values) {
 }
 $serviceNameList = $services.Keys | Sort-Object
 
-
 $toc = @()
+$corePackageItems = @()
+$coreServiceToc = @()
 foreach ($service in $serviceNameList) {
   Write-Host "Building service: $service"
 
   $packageItems = @()
 
   # Client packages get individual entries
   $clientPackages = $packagesForToc.Values.Where({ $_.ServiceName -eq $service -and ('client' -eq $_.Type) })
-  $clientPackages = $clientPackages | Sort-Object 'Package', 'GroupId' | Get-Unique
-  if ($clientPackages) {
-    foreach ($clientPackage in $clientPackages) {
+  $clientPackages = $clientPackages | Select-Object * -Unique
+  foreach ($clientPackage in $clientPackages) {
+    if ($service -eq 'Core') {
```

Comment:
As we talked earlier this is likely to change to "Other - Core" so we will need to figure out. Why do we need to separate the core packages out directly?

Reply:
I thought JS has already made this change (put core under other) and we have to follow what we have. Is this still in discussion?
@danieljurek do you know the current status? Where is the best place for 'core', parallel with other services or put under 'Other'?

Reply:
Oh, I know what you mean. Do you mean JS is able to have core under other, why we need to literally do this for Java? I am going to check why JS is able to do this but Java cannot.

Reply:
In JS csv, the service name for core is 'Other'. The right path is to change over there. Will remove the logic here

Reply:
Nesting in the `Other` service is controlled by dashes: 

If you want: 
* Other
  * Core
    * Client
      * AMQP 
      * HTTP Netty
      * etc...

Then set the service name to "Other" and the display name to (for AMQP) 

```
Core - Client - AMQP
```